README.md

```diff
@@ -522,6 +522,32 @@ export function Header({ postId }) {
 
 [`@nanostores/solid`]: https://github.com/nanostores/solid
 
+### Lit
+
+Use [`@nanostores/lit`] and `StoreController` reactive controller
+to get store’s value and re-render component on store’s changes.
+
+```ts
+import { StoreController } from '@nanostores/lit';
+import { profile } from '../stores/profile.js'
+import { Post } from '../stores/post.js'
```

Comment:
Oops, you missed a few semicolons. But in fact, we don't use them at all in the docs 😅

Reply:
That is indeed an oopsie 😅 Thanks